src/main/java/com/google/devtools/build/lib/query2/output/OutputFormatter.java

```diff
@@ -103,13 +103,21 @@ public Target apply(Node<Target> input) {
   public static ImmutableList<OutputFormatter> getDefaultFormatters() {
     return ImmutableList.of(
         new LabelOutputFormatter(false, LINE_TERM),
+        new LabelOutputFormatter(false, "\0"),
```

Comment:
This is not needed: QueryOptions are directly avaliable to `OutputFormatter#output` method, so you can just get it from there,